frame/staking/src/lib.rs

```diff
@@ -891,6 +891,10 @@ pub trait Trait: frame_system::Trait + SendTransactionTypes<Call<Self>> {
 	/// equalize will not be executed at all.
 	type MaxIterations: Get<u32>;
 
+	/// The threshold of improvement that should be provided for a new solution to be accepted.
+	// TODO: fancy a shorter name? my fingers do not like this.
+	type SolutionImprovementThreshold: Get<Perbill>;
```

Comment:
```suggestion
	type MinSolutionScoreBump: Get<Perbill>;
```
Feel it's roughly as shitty, but maybe you're going to like it more :joy: